static/sass/styles.scss

```diff
@@ -5,9 +5,8 @@ $color-tabs-active-bar: #e95420;
 // import cookie policy
 @import "@canonical/cookie-policy/build/css/cookie-policy";
 
-$color-brand: #7c355d;
+$color-brand: #e95420;
```

Comment:
I'm not sure if this matters since the homepage is being revamped as well, but FWIW this change has a pretty drastic impact on the current homepage https://canonical-com-916.demos.haus/

Reply:
> I'm not sure if this matters since the homepage is being revamped as well, but FWIW this change has a pretty drastic impact on the current homepage https://canonical-com-916.demos.haus/

I wouldn't worry about this, we are not releasing any of these new pages until the homepage is ready, so noone will ever see it.

Reply:
This is a bit out of scope, let's do it in a different PR!